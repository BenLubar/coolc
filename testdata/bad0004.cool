class Main(var x : Int) {
}
