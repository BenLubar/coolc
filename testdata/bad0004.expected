-: wrong number of method arguments
testdata/bad0004.cool:1:7: (method is declared here)
